python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -607,6 +618,13 @@ def with_run_info(
             )
         )
 
+    def with_backfill_info(self, backfill_id: str) -> "TickData":
+        return self._replace(
+            backfill_ids=[*self.backfill_ids, backfill_id]
+            if backfill_id not in self.backfill_ids
+            else self.run_keys
```

Comment:
Should `self.run_keys` be `None` instead?

Reply:
Oops, should be self.backfill_ids